src/crypto/crypto_sig.h

```diff
@@ -30,7 +30,7 @@ class SignBase : public BaseObject {
     kSignPrivateKey,
     kSignPublicKey,
     kSignMalformedSignature
-  } Error;
```

Comment:
This doesn't need to be typedef *or* using

Reply:
@addaleax I think we should we turn this into an `enum class`. A lot of its usage in `crypto_sig.cc` seems to precede the enumerations with `Error::` except for a few instances.

Reply:
Yeah, both `enum class` or plain `enum` seem fine here